src/Annotations/Serializer/CommonMark/HtmlRenderer.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace eLife\Annotations\Serializer\CommonMark;
+
+use HTMLPurifier;
+use League\CommonMark\Block\Element\AbstractBlock;
+use League\CommonMark\Environment;
+use League\CommonMark\HtmlRenderer as CommonMarkHtmlRenderer;
+
+class HtmlRenderer extends CommonMarkHtmlRenderer
```

Comment:
The name doesn't describe what it's doing (`HtmlPurifierRenderer`?). Better to use composition too.

Reply:
Misses out `renderInlines()` too?

Reply:
I was working under that assumption. I will verify it by introducing a test.